apps/server/src/modules/tool/school-external-tool/service/school-external-tool-validation.service.ts

```diff
@@ -15,13 +15,12 @@ export class SchoolExternalToolValidationService {
 	) {}
 
 	async validate(schoolExternalTool: SchoolExternalTool): Promise<void> {
-		this.commonToolValidationService.checkForDuplicateParameters(schoolExternalTool);
```

Comment:
You removed it because the commonValidationService validates this by default, right?